src/screen/PushSettingsScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {Divider, Icon, Layout, ListItem, Text} from '@ui-kitten/components';
+import GenericNavigationLayout from 'presentational/GenericNavigationLayout';
+import {DrawerNavigationProp} from '@react-navigation/drawer';
+import {StyleSheet, Switch, View} from 'react-native';
+import globalStyles, {standardPadding} from 'src/styles';
+import Padder from 'presentational/Padder';
+import {useDataContext} from 'context/DataContext';
+import {useMutation, useQuery, useQueryClient} from 'react-query';
+import LoadingView from 'presentational/LoadingView';
+import messaging from '@react-native-firebase/messaging';
+
+type PropTypes = {
+  navigation: DrawerNavigationProp<DrawerParamList>;
+};
+
+export function PushSettingsScreen(props: PropTypes) {
+  const {navigation} = props;
+  const {topics, toggleTopic, isLoading} = useTopics();
+
+  return (
+    <GenericNavigationLayout title="Push Notifications" onBackPressed={() => navigation.goBack()}>
+      <Layout level="1" style={styles.container}>
+        <Text>Hi there! To stay informed choose which Push Notifications you'd like to receive.</Text>
+        <Padder scale={2} />
+        <Divider />
+        <Padder scale={1} />
+        <View style={globalStyles.flex}>
+          {isLoading ? (
+            <LoadingView />
+          ) : (
+            topics.map((item) => (
+              <ListItem
+                key={item.id}
+                title={item.label}
+                accessoryRight={() => (
+                  <Switch value={item.selected} onValueChange={(subscribe) => toggleTopic({id: item.id, subscribe})} />
+                )}
+              />
+            ))
+          )}
+          <Padder scale={2} />
+          <Divider />
+        </View>
+        <ListItem
+          accessoryLeft={(p) => <Icon {...p} name={'info-outline'} />}
+          title={"Don't forget to enable notifications in your phone setting in the app section"}
```

Comment:
```suggestion
          title={"Don't forget to enable notifications in your phone's settings"}
```

Reply:
This only applies to IOS users, right?

Reply:
in android, it's also possible to disable notifications on the app settings. so I guess both?

Reply:
Yes I know what you mean.. What I was referring to is that on IOS the user won't get PN until it manually approves the permissions, so if the user by mistake declines the permissions (the prompt we have) the message makes sense because it helps to know where to enable it again.. that's not the case on android, so with this message a user could think that for the PN to work he/she needs to go to the settings and enable it manually

Reply:
true, I think I will revise this on https://github.com/litentry/litentry-app/issues/160